.github/workflows/build-example-web.yml

```
@@ -8,8 +8,14 @@ jobs:
     env:
       TEST_ENDPOINT: hardware-example.onekeytest.com
       PROJECT_PATH: packages/connect-examples/expo-example
+      COMMIT_SHA: ${{ github.sha }}
+      COMMIT_SHORT_SHA: ${{ github.sha }}
     steps:
       - uses: actions/checkout@v3
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)